x-pack/plugins/search_indices/public/utils/indices.ts

```diff
@@ -35,3 +35,12 @@ export function generateRandomIndexName(
 
   return result;
 }
+
+export function getFirstNewIndexName(startingIndexNames: string[], currentIndexNames: string[]) {
```

Comment:
was this added just in case or did you see clashes?

Reply:
This was added since create index can be used when other indices exist. So we needed a way to find the difference of the index names list when polling.

Thinking about it now I am worried if this will or wont catch a new index when there are a lot of them, like will new indices always be included in the first page of results?